projects/plutus/test/test_calibration.c

```diff
@@ -0,0 +1,64 @@
+#include <inttypes.h>
+
+#include "current_calibration.h"
+#include "current_sense.h"
+#include "delay.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#define TEST_MAX_CURRENT_POINT 3
+#define TEST_SAMPLE_DELAY_SECONDS 0
+
+static void prv_callback(CurrentSenseValue *value, void *context) {
+  LOG_DEBUG("Voltage =  %" PRId32 ", Current =  %" PRId32 "\n", value->voltage, value->current);
+}
+
+static CurrentCalibrationStorage s_storage = { .adc_storage = {
```

Comment:
Please fix the formatting. It usually helps to append a trailing comma (i.e. to `.filter_mode`)

Reply:
Fixed